### ISSUE TITLE
In-depth game example documentation section hard to understand

### ISSUE THREAD

**Issue by Tiling1867** [pyglet/pyglet#1027]
ATTENTION!:
If you are experiencing issues migrating from pyglet 1.5 --> 2.0, please first read the migration
section in the documentation. https://pyglet.readthedocs.io/en/latest/programming_guide/migration.html

**Describe the bug**
The **In-depth game example** documentation section is not clear. The file where the code should be written is almost never specified which makes the documentation hard to follow.

**How To Reproduce**
Go to the page [In-depth game example](https://pyglet.readthedocs.io/en/latest/programming_guide/examplegame.html) read the documentation and try to reproduce it.

**Comment by benmoran56** [comment]
Thanks for opening the ticket.
I can see your point. It's not a requirement to put the code in any specific place, but it can't hurt to be more explicit.

**Comment by Matt-Viner** [comment]
Hey @benmoran56 👋
Are you actively working on this issue? I'd be more than happy to pick this up if you're doing other things!

**Comment by caffeinepills** [comment]
> Hey @benmoran56 👋 Are you actively working on this issue? I'd be more than happy to pick this up if you're doing other things!

Feel free to work on this an open a PR. Could always use an extra hand.

**Comment by caffeinepills** [comment]
Closing as done with with 3.0
